add description of system query endpoint to dataservice documentation (#17997)
pingcap/docs: tidb-cloud/data-service-manage-endpoint.md

3. Update the default name if necessary. The newly created endpoint is added to the top of the endpoint list.
4. Configure the new endpoint according to the instructions in [Develop an endpoint](#develop-an-endpoint).

### Add a predefined system endpoint

> Data Service provides an endpoint library with a list of predefined system endpoints to reduce your effort of endpoint creation. Currently, the library only includes the `/system/query` endpoint, which offers the capability to execute any SQL statements.

Reviewer: ```suggestion
Data Service provides an endpoint library with predefined system endpoints that you can directly add to your Data App, reducing the effort in your endpoint development. Currently, the library only includes the `/system/query` endpoint, which offers the capability to execute any SQL statements.
```

> Data Service provides an endpoint library with predefined system endpoints that you can directly add to your Data App, reducing the effort in your endpoint development. Currently, the library only includes the `/system/query` endpoint, which offers the capability to execute any SQL statements.

Reviewer: ```suggestion
Data Service provides an endpoint library with predefined system endpoints that you can directly add to your Data App, reducing the effort in your endpoint development. Currently, the library only includes the `/system/query` endpoint, which enables you to execute any SQL statement by simply passing the statement in the predefined `sql` parameter.
```

To add a predefined system endpoint to your Data App, perform the following steps:

> 1. Navigate to the [**Data Service**](https://tidbcloud.com/console/data-service) page of your project.

Reviewer 2: ```suggestion
1. Navigate to the [**Data Service**](https://tidbcloud.com/console/data-service) page of your project.

```

2. In the left pane, locate your target Data App, click **+** to the right of the App name, and then click **Manage Endpoint Library**.

    A dialog for endpoint library management is displayed. Currently, only **Execute Query** (that is, the `/system/query` endpoint) is provided in the dialog.

3. To add the `/system/query` endpoint to your Data App, toggle the **Execute Query** switch to **Added**.

    > **Tip:**
    >
    > To remove an added predefined endpoint from your Data App, toggle the **Execute Query** switch to **Removed**.

4. Click **Save**.

    > **Note:**
    >
    > - After you click **Save**, the added or removed endpoint is deployed to production immediately, which makes the added endpoint accessible and the removed endpoint inaccessible immediately.

>     > - If a non-predefined endpoint with the same path and method already exists in the current app, the creation of the system endpoint will fail.

Reviewer 2: ```suggestion
    > - If a non-predefined endpoint with the same path and method already exists in the current App, the creation of the system endpoint will fail.
```

    The added system-provided endpoint is displayed at the top of the endpoint list.

5. Check the endpoint name, SQL statements, properties, and parameters of the new endpoint.

    > **Caution:**
    >
    > The `/system/query` endpoint is powerful and versatile but can be potentially destructive. Use it with discretion and ensure the queries are secure and well-considered to prevent unintended consequences.

    - Endpoint name: the endpoint name and path is `/system/query`, and the request method `POST`.

>     - SQL statements: the `/system/query` endpoint does not come with any SQL statement. You can write your desired SQL statements in the middle section of the page. Note that SQL statements for the `/system/query` endpoint are for console development and testing only and will not be saved.

Reviewer: ```suggestion
    - SQL statements: the `/system/query` endpoint does not come with any SQL statement. You can write your desired SQL statements in the SQL editor of the page. Note that SQL statements written in the SQL editor for the `/system/query` endpoint are for development and testing only and will not be saved in the endpoint configuration.
```

Reviewer: @jusong PTAL

>     - Endpoint name: the endpoint name and path is `/system/query`, and the request method `POST`.
>     - SQL statements: the `/system/query` endpoint does not come with any SQL statement. You can write your desired SQL statements in the middle section of the page. Note that SQL statements for the `/system/query` endpoint are for console development and testing only and will not be saved.
>     - Endpoint properties: unlike other custom endpoints, only the `timeout` and `max rows` properties can be customized for system endpoints.
>     - Endpoint parameters: the endpoint parameters are configured automatically and cannot be modified.

Reviewer 2: ```suggestion
    - **Endpoint name**: the endpoint name and path is `/system/query`, and the request method `POST`.
    - **SQL statements**: the `/system/query` endpoint does not come with any SQL statement. You can write your desired SQL statements in the middle section of the page. Note that SQL statements for the `/system/query` endpoint are for console development and testing only, and will not be saved.
    - **Endpoint properties**: unlike other custom endpoints, only the `timeout` and `max rows` properties can be customized for system endpoints.
    - **Endpoint parameters**: the endpoint parameters are configured automatically and cannot be modified.
```

>     - Endpoint properties: unlike other custom endpoints, only the `timeout` and `max rows` properties can be customized for system endpoints.
>     - Endpoint parameters: the endpoint parameters are configured automatically and cannot be modified.

Reviewer: ```suggestion
    - Endpoint properties: you can find the properties in the **Properties** tab of the right pane of the page. unlike other custom endpoints, only the `timeout` and `max rows` properties can be customized for system endpoints. 
    - Endpoint parameters: you can find the parameters in the **Params** tab of the right pane of the page. The parameters of the `/system/query` endpoint are configured automatically and cannot be modified. 
```

2. In the left pane, click the name of your target Data App to view its endpoints.
3. Locate the endpoint you want to rename, click **...** > **Rename**., and enter a new name for the endpoint.

> **Note:**
>

> > Sstem-provided endpoints do not support renaming.

Reviewer 2: ```suggestion
> System-provided endpoints do not support renaming.
```
